### ISSUE TITLE
No module named 'GeoApps'

### ISSUE THREAD

**Issue by Mr-Nobody-dey** [miniufo/besttracks#7]
.translate_velocity() is throwing this error
It seems that Rearth or the corresponding package is not available.-> replacing Rearth with mean earth radius works though.

translate_velocity module is available in besttracks->core.py

**Comment by schaferk** [comment]
if you are using a conda environment, you can simply install the GeoApps via
conda install GeoApps


(btk) bash-3.2$ conda list | grep geo
geos                      3.11.1               hb7217d7_0    conda-forge

**Comment by Mr-Nobody-dey** [comment]
conda install GeoApps do not work. pip install geoapps works though but it throws and error at the end of installation. Thank you for your reply.

**Comment by miniufo** [comment]
Yes, I didn't notice this.  I have removed this dependency on GeoApps and made `Rearth` as a kwarg which allow users to change freely.  Please see [here](https://github.com/miniufo/besttracks/blob/a320fe871cf7679a51baa3c609167e850a85562b/besttracks/core.py#L76).

Thanks for pointing out this.
